scripts/within_tissue_analysis.py

```diff
@@ -41,8 +41,31 @@
 parser.add_argument('-f', '--alt_folder', default='Auto',
                     help='location to save')
 
+parser.add_argument('-x', '--x_matrix', default=None,
+                    help='Filename of features to use in model')
+parser.add_argument( '--filename_mut', default=None,
+                    help='Filename of sample/gene mutations to use in model')
+parser.add_argument( '--filename_mut_burden', default=None,
```

Comment:
it looks like these filename flags are repeated often. Can we make a separate file like `tcga_util.get_args`?